src/TouristInfoCards.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  let TouristInfo = [
```

Comment:
The cards are good and work correctly, however it would be nice if the image, summary and destination matched the links!